pnpm-lock.yaml

```diff
@@ -1584,28 +1584,31 @@ packages:
       buffer: 6.0.3
     dev: true
 
-  /@prisma/studio-pcw/0.459.0:
+  /@prisma/studio-pcw/0.459.0_839d6f5e1e290969a8d17594ebaffe12:
```

Comment:
Uh, what is this change about?

Reply:
It's a pnpm thing, looks like it might be linked to the `link` additions like
```
      '@prisma/sdk': link:packages/sdk
```

Maybe a new pnpm version